lib/Backend/Lower.cpp

```diff
@@ -17931,6 +17934,70 @@ Lowerer::GenerateFastInlineArrayPop(IR::Instr * instr)
     GenerateHelperToArrayPopFastPath(instr, doneLabel, bailOutLabelHelper);
 }
 
+void
+Lowerer::GenerateFastInlineHasOwnProperty(IR::Instr * instr)
+{
+    Assert(instr->m_opcode == Js::OpCode::CallDirect);
```

Comment:
Could you add a comment at the top of this function outlining the inline code? (Similar to how e.g. GenerateFastCondBranch has it)